nextflow.config

```diff
@@ -362,10 +365,10 @@ manifest {
     homePage        = 'https://github.com/Ferlab-Ste-Justine/Post-processing-Pipeline'
     description     = """Variant analysis for genome and exome GVCFs"""
     mainScript      = 'main.nf'
-    nextflowVersion = '!>=23.04.0'
```

Comment:
Is it possible to write the version on the fly ?  Or maybe just get rid of it in the manifest part and put the nextflow version in metadata...

Reply:
For info, this change was actually done automatically by nf-core when using command `nf-core bump-version --nextflow 23.10.1`.